### ISSUE TITLE
Firecracker exits immediately

### ISSUE THREAD

**Issue by andys** [nanovms/ops-documentation#260]
Hi! I'm trying to follow simple demo with static golang httpserver binary on a fresh Ubuntu 20.04 Server host.

* Firecracker works on this host with standard Alpine linux image.
* Nanos/Ops works on this host (ops run) with this binary.
* But attempting to use firecracker with the built images fails instantly, the logs shows:

```
Running Firecracker v0.24.6
2021-10-15T10:25:26.503641868 [vm0001:main:INFO:src/vmm/src/device_manager/mmio.rs:394] Artificially kick devices.
2021-10-15T10:25:26.503694214 [vm0001:main:INFO:src/vmm/src/device_manager/mmio.rs:414] kick block rootfs.
2021-10-15T10:25:26.503703720 [vm0001:main:INFO:src/vmm/src/device_manager/mmio.rs:425] kick net eth0.
2021-10-15T10:25:26.503761131 [vm0001:main:INFO:src/firecracker/src/main.rs:336] Successfully started microvm that was configured from one single json
2021-10-15T10:25:26.503844464 [vm0001:fc_vcpu 0:INFO:src/vmm/src/vstate/vcpu/mod.rs:461] Received KVM_EXIT_SHUTDOWN signal
2021-10-15T10:25:26.503880756 [vm0001:main:INFO:src/vmm/src/lib.rs:348] Vmm is stopping.
```

I tried old version of Firecracker (0.22) as well as old version of nanos (0.1.26) but same problem.

This is with firecracker config similar to the tutorial, "kernel_image_path" is the `.ops/0.1.36/kernel.img` file, and root device drive "path_on_host" is the `.ops/images/helloworlddemo.img` file.

**Comment by andys** [comment]
> i can repro this - not sure if it's a nanos problem (very well could be since it's been a while since people have asked about firecracker) or something else - regardless we'll take a look

Thanks eyberg!  Meanwhile I have noticed that golang + nanos boots quite fast in qemu, which is great.  The only remaining downside for me is the memory usage, which is still higher with qemu -  firecracker is good at running a large number of idle processes, so I think is still worth getting it working.

**Comment by sanderssj** [comment]
Will be fixed by nanovms/nanos#1610
